CONTRIBUTING.md

```diff
@@ -36,6 +37,12 @@ with as much detail as possible, including things like operating system or anyth
 The MongoDB CLI project welcomes all contributors and contributions regardless of skill or experience level.
 If you are interested in helping with the project, please follow our [guidelines](#code-contribution-guidelines).
 
+## Autoclose stale issues and PRs
+
+- After 30 days of no activity (no comments or commits are on an issue/PR) we automatically tag it as "stale" and add a message: ```This issue/PR has gone 30 days without any activity and meets the project's definition of "stale". This will be auto-closed if there is no new activity over the next 60 days. If the issue is still relevant and active, you can simply comment with a "bump" to keep it open, or add the label "not_stale". Thanks for keeping our repository healthy!```
```

Comment:
```suggestion
- After 30 days of no activity (no comments or commits on an issue/PR) we automatically tag it as "stale" and add a message: ```This issue/PR has gone 30 days without any activity and meets the project's definition of "stale". This will be auto-closed if there is no new activity over the next 60 days. If the issue is still relevant and active, you can simply comment with a "bump" to keep it open, or add the label "not_stale". Thanks for keeping our repository healthy!```
```